### PR TITLE
main/gnome-remote-desktop: new package (46.3)

### PR DESCRIPTION
**Pull request by triallax**
- **main/sdl_ttf: move from contrib**
- **main/uriparser: move from contrib**
- **main/pcsc-lite: move from contrib**
- **main/ffnvcodec-headers: new package (12.2.72.0)**
- **main/freerdp3: new package (3.6.3)**
- **main/gnome-remote-desktop: new package (46.3)**
- **main/gnome: add gnome-remote-desktop to -apps**

### CODE REVIEW

**Comment by triallax** [comment]
haven't really figured out how i'm supposed to test this yet, so testing welcome

File: main/ffnvcodec-headers/template.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+pkgname = "ffnvcodec-headers"
```
Comment (q66) [inline main/ffnvcodec-headers/template.py:1]: what's up with this scuffed name

why not just nv-codec-headers

Comment (triallax) [reply main/ffnvcodec-headers/template.py:1]: idk it's been a while, i don't remember why i chose this name

Comment (nekopsykose) [reply main/ffnvcodec-headers/template.py:1]: probably from copying another distro and cause i called it that

Comment (triallax) [reply main/ffnvcodec-headers/template.py:1]: it's all YOUR fault!!11!!!1

Comment (nekopsykose) [reply main/ffnvcodec-headers/template.py:1]: 😔

**Comment by NovaAndrom3da** [comment]
Fixes #1878

**Comment by triallax** [comment]
can't really be bothered to carry this forward, others are welcome to revive the pr if so they wish

**Pull request closed without merging**